Cable

comma opened the pull request:
Makes cable use BYOND directions
Instead of 11,12 they use 32,16 for DOWN,UP
Merges multiz handling into main logic

PsiOmegaDelta commented:
Logic and reasonable values in an SS13 codebase? Heresy.

PsiOmegaDelta commented:
#15883 is a little bit of a headache with what should come first.

comma commented:
I'm fine with #15883 being first, resolving will be ez pz since I just need search&replace in text editor.

comma commented:
Seems like it fixes 'multiz cables not connecting' part of #15558

mwerezak commented:
Yeah, that power_list() stuff seems kind of sketchy. Plus using reverse_dir[d] seems like a way better means to handle reversing direction.

comma committed:
Makes cable use BYOND directions

Instead of 11,12 they use 32,16 for DOWN,UP
Merges multiz handling into main logic
Updates maps
